src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -150,9 +150,9 @@
 	<PackageReference Include="SharpDX.Direct3D9" Version="4.2.0" />
 	<PackageReference Include="SharpDX.DXGI" Version="4.2.0" />
 	<PackageReference Include="SharpDX.Mathematics" Version="4.2.0" />
-    <Reference Include="Dynamo.Microsoft.Xaml.Behaviors, Version=1.1.0.0, Culture=neutral, processorArchitecture=MSIL">
+    <Reference Include="Dynamo.Microsoft.Xaml.Behaviors, Version=1.1.39.4761, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
so this version of Microsoft.Xaml.Behaviors does not target net6 - but their latest source does - would it be beneficial to build from source and run that assembly through the rename application? Which target framework did you actually convert here for net6?

Reply:
I used the latest nuget, which targets net5-windows. I could build it from source. Weird thing is that that repo has a nuget publish git-action, but I cannot see wehere it is published (probably a microsoft internal server)

I am also not sure if we will be able to apply this strategy for all our packageReferences. 
There are braking changes between net5 and net6, but some packages target an older version of dotnet intentionally just so it will be compatible with more .net runtimes (that's what I did for forge units for example). 

For some we may just have to use a net5 target and just test it on our app + net6